services/graphql/src/graphql/definitions/app-user.js

```diff
@@ -3,11 +3,12 @@ const gql = require('graphql-tag');
 module.exports = gql`
 
 extend type Query {
-  appUsers(input: AppUsersQueryInput!): AppUserConnection! @requiresAppRole
+  appUsers(pagination: PaginationInput = {}, sort: AppUserSortInput = {}): AppUserConnection! @requiresAppRole
```

Comment:
The `pagination` and `sort` args should be combined into a single input. See
https://github.com/base-cms/id-me/blob/1d512f268b539fdde983aace151655810edb7dd6/services/graphql/src/graphql/definitions/team.js#L7
And
https://github.com/base-cms/id-me/blob/1d512f268b539fdde983aace151655810edb7dd6/services/graphql/src/graphql/definitions/team.js#L60